pkg/sources/gcs/gcs.go

```diff
@@ -107,6 +112,12 @@ func configureGCSManager(aCtx context.Context, conn *sourcespb.GCS, concurrency
 		gcsManagerAuthOption = withDefaultADC(aCtx)
 	case *sourcespb.GCS_Unauthenticated:
 		gcsManagerAuthOption = withoutAuthentication()
+	case *sourcespb.GCS_Oauth:
+		oauth2Client, err := oauth2Client(aCtx, conn.GetOauth())
```

Comment:
nit: Having the variable and function name the same could cause confusion.

Reply:
ooof, great call.